packages/frontend-2/components/settings/user/profile/DeleteAccountDialog.vue

```diff
@@ -42,15 +39,14 @@
         </FormButton>
       </div>
     </form>
-  </LayoutDialogSection>
+  </LayoutDialog>
 </template>
 <script setup lang="ts">
-import { LayoutDialogSection } from '@speckle/ui-components'
-import { TrashIcon, ExclamationTriangleIcon } from '@heroicons/vue/24/outline'
+import { ExclamationTriangleIcon } from '@heroicons/vue/24/outline'
 import { useForm } from 'vee-validate'
 import type { GenericValidateFunction } from 'vee-validate'
 import { graphql } from '~~/lib/common/generated/gql'
-import type { UserProfileEditDialogDeleteAccount_UserFragment } from '~~/lib/common/generated/gql/graphql'
+import type { User } from '~~/lib/common/generated/gql/graphql'
```

Comment:
You're asking for way too much when requiring the full User type, making it practically impossible to actually use these components without overriding/ignoring TS errors, use fragments to define specific requirements -
https://www.notion.so/speckle/General-Web-Code-Review-Guide-1f2bf3dfea634460b3778ac3cf4aa1bb?pvs=4#d40e40efc32f4e5ea67b68e848aadcd0